hardhat.config.ts

```diff
@@ -1,16 +1,29 @@
 import { HardhatUserConfig } from "hardhat/config";
 import "@nomicfoundation/hardhat-toolbox";
 
-const config: HardhatUserConfig = {
-  solidity: {
+const solidityProfiles = {
+  default: {
     version: "0.8.16",
     settings: {
       optimizer: {
         enabled: true,
         runs: 20000
       },
-    },
+    }
   },
+  sec_council_mgmt: {
+    version: "0.8.16",
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 200
```

Comment:
Can we bump to 2k?

Reply:
seems like nominee deploy lib is too big with 2k